MarketplaceServiceTests/Repository/MarketplaceRepositoryTests.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using marketplaceservice.DatastoreSettings;
+using marketplaceservice.Domain;
+using marketplaceservice.Repositories;
+using Mongo2Go;
+using Xunit;
+
+namespace marketplaceservicetests.Repository
+{
+    // : IDisposable
```

Comment:
Weghalen

Reply:
Welke van de 4 lines? Als je een suggest gebruikt is het misschien iets duidelijker ;)